src/main/java/sirius/db/jdbc/batch/BatchContext.java

```diff
@@ -101,8 +102,9 @@ protected Connection getConnection(String realm) {
 
     protected Connection createConnection(String realm) {
         try {
-            Connection connection = oma.getDatabase(realm).getLongRunningConnection();
-            changeAutoCommit(connection, false, false);
+            Database database = oma.getDatabase(realm);
+            Connection connection = database.getLongRunningConnection();
+            changeAutoCommit(connection, database.isAutoCommit(), false);
```

Comment:
shouldn't we use a Capability like database.hasCapability(TRANSACTIONS) or the like and disable this for the CH driver?